wesnoth-help.po

```diff
@@ -3976,9 +3975,9 @@ msgid ""
 msgstr ""
 "\n"
 "\n"
-"Mezi základní typy terénu patří Pláně, Hory, Písky, Voda a Bažiny. Mimo "
-"těchto pár základních existuje ještě několik dalších - seznam terénů, které "
-"jste už viděli, můžete najít na konci této stránky.\n"
+"Mezi základní typy terénu patří například Plochý terén, Kopce, Hory, Písky, "
+"Voda a Bažiny. Mimo těchto pár základních existuje ještě několik dalších - "
```

Comment:
Juknul jsem na to odpoledne a pak to na běžkách rozmýšlel. Jsem rád, že to opravujete a věřím, že k lepšímu (nechci to kontrolovat). Jen co se týče těch velkých písmen, nebál bych se jich zbavit. Vlevo v nápovědě, kde je název terénu, má velké písmeno své místo, ale ne v textu, když to jsou pořád ty samé písky a hra nemá žádné Písky písků, co by potřebovaly odlišit nebo zdůraznit. Už to, že se píše o terénu vylučuje jakoukoli mýlku, takže z mého pohledu tam to velké písmeno nemá co dělat.
```suggestion
"Mezi základní typy terénu patří například plochý terén, kopce, hory, písky, "
"voda a bažiny. Mimo těchto pár základních existuje ještě několik dalších – "
```